resources/i18n/en-US/builtin.ftl

```diff
@@ -0,0 +1,3 @@
+app-name = Druid
```

Comment:
I recommend using Fluent Terms for that - https://projectfluent.org/fluent/guide/terms.html